README.md

```diff
@@ -62,6 +62,9 @@ If you're using Bundler, you can use the
 [`force_ruby_platform`](https://bundler.io/v2.3/man/gemfile.5.html#FORCE_RUBY_PLATFORM)
 option in your Gemfile.
 
+Note that Windows builds may fail with [`undefined reference` errors](https://github.com/pkgconf/pkgconf/issues/322)
```

Comment:
Does this only affect people who are trying to compile Abseil on Windows (so users of the precompiled gem or anyone who is compiling against a system RE2 are unaffected)?

Reply:
Yes, I added a clarifying statement.